packages/app/src/Charcoal/App/ServiceProvider/AppServiceProvider.php

```diff
@@ -4,14 +4,17 @@
 
 // From PSR-7
 use Charcoal\Factory\GenericResolver;
+use phpDocumentor\Reflection\Types\Void_;
```

Comment:
PHPStorm has snuck in an external/dev import.

```suggestion
```

Reply:
👍